.github/workflows/test.yml

```diff
@@ -64,3 +65,14 @@ jobs:
           name: JEST Tests
           path: 'reports/jest-*.xml'
           reporter: jest-junit
+
+      - name: GitHub Action self-test
+        id: self_test
+        uses: ./
+        with:
+          server: ${{ env.SERVER_URL }}
+          api_key: ${{ env.ADMIN_API_KEY }}
```

Comment:
Let's make sure we're using the action in the same way we're espousing in the README; environment variables.

Reply:
Sure, will tweak